qiita_pet/util.py

```diff
@@ -38,9 +40,12 @@ def linkify(link_template, item):
     """
     return link_template.format(*item)
 
+# global compiled regex so compiled once and super fast after that
+clean_str_regex = re.compile("(\.|:)")
```

Comment:
These are capturing parentheses. Although this won't make the `clean_str` function fail, it would be better to use a character class:

```python
clean_str_regex = re.compile("[.:]")
```